release-notes/5.0/api-diff/netcoreapp3.1/.Net/standalone/5.0-standalone_System.Diagnostics.DiagnosticSource.md

```diff
@@ -0,0 +1,224 @@
+# System.Diagnostics.DiagnosticSource
```

Comment:
@Anipik does this means the whole System.Diagnostics.DiagnosticSource library was not included in 3.1 and it is now included in 5.0? 

cc @noahfalk

Reply:
this means that these apis were added in the package. The shared framework api changes are in separate markdown diff

Reply:
@Anipik Is it first time we create such package for System.Diagnostics.DiagnosticSource.md? I am seeing the diff here is just including all APIs there even the APIs we didn't touch for many years.

Reply:
i resolved the issue, it was picking the assembly from shared framework. i removed the packages which are part of the shared framework